packages/server-bridge/src/bridge.js

```diff
@@ -353,6 +374,34 @@ function init({
       });
   }
 
+  async function onHttpRequest(req, res) {
+    const { body: { body }, params, user } = req;
```

Comment:
@sloops77 notice there is no validation for the `user` property inside the request, not sure if that should be dealt with here...